apps/wing-console/console/ui/src/layout/vscode-layout.tsx

```diff
@@ -48,12 +49,19 @@ export const VscodeLayout = ({ cloudAppState, wingVersion }: LayoutProps) => {
     logsRef,
     logs,
     onResourceClick,
+    wingfile,
   } = useLayout({
     cloudAppState,
     defaultLogLevels: ["info", "warn", "error"],
   });
 
-  const { title } = useContext(AppContext);
+  const title = useMemo(() => {
+    return `${wingfile.data ?? ""}`;
+  }, [wingfile.data]);
+
+  useEffect(() => {
+    document.title = title ? `${title} | Wing Console` : "Wing Console";
```

Comment:
haven't we discussed to follow vscode convention? fileName - Wing Console

Reply:
true, fixed